client/app/dbaas/logs/detail/options/home/logs-options-home.html

```diff
@@ -13,7 +13,7 @@
     <oui-spinner data-ng-show="ctrl.currentOptions.loading || ctrl.selectedOffer.loading"></oui-spinner>
     <div class="row mb-4" data-ng-if="!ctrl.isBasicOffer() && !ctrl.selectedOffer.loading">
         <p class="oui-paragraph col-md-12" data-translate="log_options_current_offer_title"></p>
```

Comment:
There's no need for a `col-md-12`

Reply:
Have replaced `col-md-12` with `margin left`